internal/cli/import_roles.go

```diff
@@ -0,0 +1,221 @@
+package cli
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/auth0/auth0-cli/internal/auth0"
+	"github.com/auth0/go-auth0/management"
+)
+
+type Role = management.Role
+
+// Put here the Roles handler logic
+
+// 1. Fetch all existing items
+// 2. Map all YAML items to the corresponding Go SDK models
+// 3. Distribute in create, delete, update and conflicts maps
+// 4. Perform patch diff
+// 5. Execute creates, updates and deletes
+
+// 1. Fetch all existing items
+
+func (cli *cli) getAllRoles(context context.Context) ([]*Role, error) {
+	list, err := getWithPagination(
+		context,
+		0, // Get *all* roles
+		func(opts ...management.RequestOption) (result []interface{}, hasNext bool, apiErr error) {
```

Comment:
looks like `opts` is available with the context that is passed in?

Reply:
I guess I'm curious where opts is passed in to `cli.getAllAPIs(ctx)` since only a ctx is passed in?

Reply:
Oh, the func takes opts but none are passed of course. This is copypasta from https://github.com/auth0/auth0-cli/blob/main/internal/cli/organizations.go#L688.